src/styles/molecules/_tabs.scss

```diff
@@ -0,0 +1,34 @@
+.tab{
+  width: 40%;
+  display: inline-block;
+}
+.tab__nav ul{
```

Comment:
If you remove the `<ul>` this should only be `.tab__nav {`.

Then you can remove all the styles that reset the behaviours of `ul`